### ISSUE TITLE
Main Telco architecture diagram: enhancement request for observability depictions

### ISSUE THREAD

**Issue by meangreenmachine** [microsoft/industry#155]
**Describe the bug**

Great to see Azure Monitor Log Analytics integration. Should we show the equivalent Monitor Metrics visualization and analysis e.g. [Metrics Explorer](https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-charts)? The observability story may feel incomplete if logs visualization and analysis is depicted, but not metrics.

Furthermore, for the Microsoft 1P packet core, we will likely also leverage Azure Data Lake Storage (ADLS) fronted by Azure Data Explorer (ADX) for application service level analytics (e.g. Telco subscriber analytics). It might be good to show these components near the PowerBI and Storage Account icons in [the main Telco Architecture Diagram](https://github.com/microsoft/industry/blob/main/telco/docs/telco-industry-reference-architecture.png).

**Steps to reproduce**

N/A

**Screenshots**

View the diagram in question by [clicking here](https://github.com/microsoft/industry/blob/main/telco/docs/telco-industry-reference-architecture.png)

**Comment by rcarmo** [comment]
I agree with this. I was actually searching for an analytics specific issue, but in the telco industry there is a clear need to explain how we tackle network analytics and observability in a distinct way from how we do, say, customer analytics (which is a more cross-industry thing, even though it is critical to telcos from a user experience perspective).

**Comment by victorar** [comment]
Hi @meangreenmachine, @rcarmo 

Sorry for the delay with providing feedback on this suggestion.

Would the Network Analytics Landing Zone, which is being developed in partnership with AfO team, would address the asks above?

See here details for Network Analytics Landing Zone: https://github.com/microsoft/industry/blob/main/telco/solutions/observability/readme.md
